general/github/readmes/README_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+# name-of_repo
```

Comment:
I used the tps_ui README as a template for the template, if we need to add or remove anything please leave a comment!